packages/theme/pages/Checkout/Shipping.vue

```diff
@@ -225,6 +235,7 @@
       <VsfShippingProvider
         v-if="isFormSubmitted"
         @submit="router.push(localePath({ name: 'billing' }))"
+        @back="() => isFormSubmitted = !isFormSubmitted"
```

Comment:
Just a thought - would it make sense to create a handler in the setup() to toggle the form editability here? To separate the concerns. Additionally, renaming isFormSubmitted to isFormEditable (it would require negating already used isFormSubmitted - so !isFormSubmitted -> !!isFormEditable -> isFormEditable; isFormSubmitted -> !isFormEditable).

Reply:
For now I will let it be as it is since i think that submission of the form is tightly tied with the process